js/hexaworld.js

```diff
@@ -0,0 +1,229 @@
+var hexaworld = {
+
+  $schema: "http://json-schema.org/schema#",
+  id: "https://github.com/hexaworld/hexaworld-schema/schema.js",
+
+  definitions: {
+
+    // basic definitions
+
+    pixelCoord: {
```

Comment:
Not convinced yet because we do have two distinct coordinate spaces, and two different ideas of position. You don't think a "position" can be either in axial coords or pixel coords?